src/integrationTest/schema/schema.test.ts

```diff
@@ -65,47 +64,20 @@ describe('Sam Schema Regression', function () {
     })
 })
 
-describe.skip('getDefaultSchemas()', () => {
+describe('getDefaultSchemas()', () => {
     beforeEach(async () => {})
 
-    it('uses cache on subsequent request for CFN/SAM schema', async () => {
+    it('uses cache after initial fetch for CFN/SAM schema', async () => {
         await getDefaultSchemas()
-
-        // IMPORTANT: Since CFN and SAM use the same schema, the order their schema is retrieved is irrelevant
-
-        // Schema is downloaded on initial retrieval
-        assertTelemetry(
-            'toolkit_getExternalResource',
-            { url: samAndCfnSchemaUrl, passive: true, result: 'Succeeded' },
-            0
-        )
-        // Schema is retrieved from cache on subsequent retrieval
-        assertTelemetry(
-            'toolkit_getExternalResource',
-            { url: samAndCfnSchemaUrl, passive: true, result: 'Cancelled', reason: 'Cache hit' },
-            1
-        )
-    })
-
-    it('uses cache for all requests on second function call', async () => {
```

Comment:
Don't need this 2nd test because we can test metrics order in one test.

However, a future TODO is to delete the cached item in globalStorage so that the test works better for local development (not only CI).